code-studio/src/js/components/share_dialog_body.jsx

```diff
@@ -92,6 +92,16 @@ var ShareDialogBody = React.createClass({
         appType={this.props.appType}/>;
     }
 
+    var exportButton = null;
+    if (this.props.appType === 'applab') {
+      // TODO: Can I not proliferate the use of global references to Applab somehow?
```

Comment:
I believe the React way to do this would be to pass the export callback as a prop.